types/opt/bool.go

```diff
@@ -18,6 +18,16 @@ import (
 // field without it being dropped.
 type Bool string
 
+// NewBool constructs a new Bool value equal to b. The returned Bool is set,
+// unless Set("") or Clear() methods are called.
+func NewBool(b bool) Bool {
+	if b {
```

Comment:
Can this function be simplified to look more like `Set`, below? I.e. use `strconv`?

Reply:
Duh, yeah that's cleaner!